src/databricks/labs/ucx/source_code/python_ast_util.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+import ast
+from dataclasses import dataclass
+
+
+@dataclass
+class Span:
```

Comment:
LSP calls this datasctructure as `Range`.

Reply:
Ok, I tried to use it but it triggers a cyclic import, because lsp imports languages which imports my linter.

I think these foundational utilities should be importing much (if anything) from the codebase, instead they'd be at the bottom of the import chain (everything depends on them, they depend on nothing).